changes/v5.0.13-en.md

```diff
@@ -0,0 +1,7 @@
+# v5.0.13
+
+## Enhancements
+
+- Add `limiter` update API [#9133](https://github.com/emqx/emqx/pull/9133).
```

Comment:
should be a part of 5.0.12

Reply:
sorry, this should not be a part of the fix.

Reply:
✅